pl_bolts/callbacks/data_monitor.py

```diff
@@ -20,15 +19,14 @@
     warn_missing_pkg("wandb")
 
 
-@under_review()
 class DataMonitorBase(Callback):
 
     supported_loggers = (
         TensorBoardLogger,
         WandbLogger,
     )
```

Comment:
Btw, looking at the tests, could we also support `LoggerCollection` by some sort of iteration over included `Logger`s?

Reply:
Yep, ill check Comet 👍 
But wasn't `LoggerCollection` deprecated? I think we can accept just a list of Loggers. Should check a bit more

Reply:
You're right, `LoggerCollection` got depreciated in the meantime. Supporting a list of `Logger`s is a nice addition, but I feat it might be a bit more difficult, so if it's too much, that by itself is not a priority.